src/index.js

```diff
@@ -61,14 +58,13 @@ function organizeImports(unsortedCode, extension) {
 const parsers = {
   typescript: {
     ...typescriptParsers.typescript,
-    preprocess(text) {
-      return organizeImports(text, '.ts')
+    preprocess(text, opts) {
+      return organizeImports(text, '.ts', opts.filepath)
```

Comment:
Since `import-sort-config` is expecting an extension and a directory, not the path to the file we're working with, maybe this should be:
```suggestion
      return organizeImports(text, path.extname(opts.filepath), path.dirname(opts.filepath))
```

It does seem to work for me as-is, though (after fixing the syntax issue)